lcls_tools/cor_plot/README.MD

```diff
@@ -0,0 +1,44 @@
+This utility can take a cor plot .mat file and turn it into a python data object.  The goal is to present the data from a cor plot in a meaningful way.This utility has a couple of test files that are used for testing, but we can use them for examples.  All examples are run from this directory.  Current example is using Python2.7
+
+Example: 'test_scan.mat'
+
+Import and Initialize cor plot data object
+ ```
+>>> from cor_plot_mat_scan import CorPlotMatScan as CPMS
+>>> cpms = CPMS('test_scan.mat')
+```
+
+Access Properties.  Here are some of the properties that are defined for the data object.
+```
+>>> cpms.fields
+('accelerator', 'status', 'ctrlPV', 'beam', 'profPV', 'ts', 'config')
+>>> cpms.accelerator
+'LCLS2'
+>>> cpms.ctrl_pv
+'SOLN:GUNB:212:BCTRL'
+>>> cpms.iterations
+10
+>>> cpms.ctrl_vals
+[0.072999999999999995, 0.073888888888888879, 0.074777777777777776, 0.07566666666666666, 0.076555555555555557, 0.077444444444444441, 0.078333333333333338, 0.079222222222222222, 0.080111111111111119, 0.081000000000000003]
+>>> cpms.beam_names
+('profx', 'xStat', 'xStatStd', 'profy', 'yStat', 'yStatStd', 'profu', 'uStat', 'uStatStd', 'method', 'stats', 'statsStd')
+>>> cpms.timestamp
+737730.05017097027
+>>> cpms.samples
+2
+```
+
+Access Beam Data.  This is very tricky and could use some cleanup.  This is all the metadata associated with the scan and goes by the nature of cpms.beam[iteration][sample][fit].  The fits are in the following order FIT = ['Gaussian', 'Asymmetric', 'Super', 'RMS', 'RMS cut peak', 'RMS cut area', 'RMS floor'].
+
```

Comment:
I also want to add a second ping/flag to this structure for cleanup later: cpms.beam[iteration][sample][fit].